core/notfound.ml

```diff
@@ -53,6 +53,15 @@ end
 module List =
 struct
   include List
+
+  let find2 p l =
+    find p l
+
+  (* add for old OCaml < 4.0.5 *)
```

Comment:
Is this still needed given that Links 0.7.3 requires OCaml > 4.0.5?

Reply:
Can also be deleted now.